server/models/DogModel.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+const locationSchema = require('./locationSchema');
+
+const dogSchema = new mongoose.Schema({
+  _id: Number,
+  name: String,
+  breed: String,
+  mainImageUrl: String,
+  imageUrls: [String],
+  energy: String, // low, medium, high
+  size: String, // small, medium, large
+  isDogFriendly: String,
```

Comment:
I'm not sure how Jacob is planning to set up the form for initial sign-up, but wondering if this could be Boolean? Same comment for isHumanFriendly

Reply:
could go either way but i think it said boolean in the dog controller.  I think boolean might be slightly easier but only by a small margin

Reply:
Boolean just "feels" better even though either case the amount of code needed client-side would only differ by a few chars, maybe 1-2 lines. But yeah, I'm on the Boolean Bandwagon, too.